lib/marcel/mime_type/definitions.rb

```diff
@@ -42,3 +42,13 @@
 Marcel::MimeType.extend "application/vnd.adobe.flash.movie", magic: [[0, "FWS"], [0, "CWS"]], extensions: %w( swf )
 Marcel::MimeType.extend "application/sql", extensions: %w( sql )
 Marcel::MimeType.extend "text/vcard", magic: [[0, "BEGIN:VCARD"]], extensions: %w( vcf ), parents: "text/plain"
+
+Marcel::MimeType.extend(
+  "application/vnd.ms-access",
+  extensions: %w( mdb accdb ),
```

Comment:
@gmcgibbon I missed this - should `mde` and `accde` be in this list too? They're the compiled versions of `mdb` and `accdb` files.

Reply:
We could add the extensions, but it also depends what the hexdumped contents are for magic detection. I'll try playing with a trial of MS access to see if I can get more fixtures to test out.